scripts/check_md5sums.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ] || ! { [ "$1" = "zson" ] || [ "$1" = "bzson" ]; }; then
+  echo 'Must specify output format to be checked: "zson" or "bzson"'
+  exit 1
+fi
+
+if [[ $(type -P "gzcat") ]]; then
+  ZCAT="gzcat"
+elif [[ $(type -P "zcat") ]]; then
+  ZCAT="zcat"
+else
+  echo "gzcat/zcat not found in PATH"
+  exit 1
+fi
+
+if [[ $(type -P "md5sum") ]]; then
+  SUMTOOL="md5sum"
+elif [[ $(type -P "md5") ]]; then
+  SUMTOOL="md5 -q"
+else
+  echo "md5sum/md5 not found in PATH"
+  exit 1
+fi
+
+ZNG_TYPE="$1"
+REPO_DIR="$(cd "$(dirname "$0")" && pwd)/.."
+cd "$REPO_DIR"
+TMPFILE=$(mktemp)
+
+for FILE in "$REPO_DIR"/"$ZNG_TYPE"/*
+do
+  COMPARE_TO="$(basename "$FILE")"
+  ZPATH=${COMPARE_TO/.${ZNG_TYPE}.gz/}
+  echo -n "${ZPATH}:" | tee -a "$TMPFILE"
+  "$ZCAT" zeek-default/"$ZPATH".log.gz \
+      | zq -f "$ZNG_TYPE" - \
+      | $SUMTOOL \
```

Comment:
Does shellcheck complain about this?

Reply:
No complaints. In fact, I'd initially put quotes around it precisely because I assumed shellcheck would complain, but then when it actually needs to use the `md5 -q` variant, you get:

```
# ./check_md5sums.sh zson
capture_loss:./check_md5sums.sh: line 40: md5 -q: command not found
```

So this is actually the one time we _want_ it to do the non-quoted expansion thing. Whether shellcheck was smart enough to pick up on that context and that's why it keeps quiet, I cannot say. 🤷‍♂